src/main/java/com/faforever/client/mod/ModManagerController.java

```diff
@@ -116,14 +149,7 @@ private void loadActivatedMods() {
   }
 
   private void filterModList() {
-    modVersionFilteredList.setPredicate(viewToggleGroup.getSelectedToggle() == uiModsButton ? UI_FILTER : SIM_FILTER);
-    modVersionFilteredList.forEach(modVersion -> {
-      if (selectedMods.contains(modVersion)) {
-        modListView.getSelectionModel().select(modVersion);
-      } else {
-        modListView.getSelectionModel().clearSelection(modListView.getItems().indexOf(modVersion));
-      }
-    });
+    onTextChange(modSearchTextField.getText());
```

Comment:
This could have stayed the same except for the assignment of the predicate and then just make this method invoked on invalidation of the text field.